test/blackbox-tests/test-cases/pkg/ocamlformat-dev-tool.t

```diff
@@ -0,0 +1,487 @@
+Exercises end to end, locking and building ocamlformat dev tool.
```

Comment:
Should "dev tool" here be hyphenated like the others "dev tool," and should OCamlFormat be like the proper noun or `ocamlformat` here?

Reply:
dev-tool would be correct, but I think it is "ocamlformat dev-tool" here.  What do you think @Leonidas-from-XIV.

Reply:
I go for `OCamlForamt` dev-tool.